devex/internal/pkg/rollout/machineconfigpool.go

```diff
@@ -97,6 +101,20 @@ func WaitForAllMachineConfigPoolsToComplete(cs *framework.ClientSet, timeout tim
 	return err
 }
 
+// Waits for the MachineConfigPool to complete. This does not consider
```

Comment:
nice!

Reply:
Thanks! A lot of the code that these specific helpers use was either written for our e2e test suite -or- was written here and then used in our e2e test suite. There's the potential for some future deduplication here.